eng/ci/templates/official/jobs/run-non-e2e-tests.yml

```diff
@@ -43,6 +43,24 @@ jobs:
       keyVaultName: azure-functions-host-$(LeaseBlob)
       secretsFilter: '*'
 
+  - task: AzureCLI@2
+    displayName: 'Azure CLI '
+    inputs:
+      azureSubscription: Azure-Functions-Host-CI-internal
+      addSpnToEnvironment: true
+      scriptType: pscore
+      scriptLocation: inlineScript
+      inlineScript: |
+        Write-Host "##vso[task.setvariable variable=ARM_CLIENT_ID]$servicePrincipalId"
+        Write-Host "##vso[task.setvariable variable=ARM_ID_TOKEN]$idToken"
```

Comment:
For the token, should we add `;issecret=true` as described [here](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/set-secret-variables?view=azure-devops&tabs=yaml%2Cbash#set-secret-variable-in-a-script-with-logging-commands)? This may already be caught by general ADO sanitization logic, but it would still be good to include regardless.

Reply:
Discussed. Already being treated as secrets. Will merge to unblock and follow up.

Reply:
ADO seems to be handling this correctly already, so this does not seem to be needed.